aws_quickstart/datadog_agentless_scanning.yaml

```diff
@@ -977,7 +980,6 @@ Resources:
           import urllib.parse
 
           LOGGER = logging.getLogger()
-          LOGGER.setLevel(logging.INFO)
```

Comment:
Following the [AWS recommendation](https://docs.aws.amazon.com/lambda/latest/dg/python-logging.html#python-logging-lib):

> If you set your function's log format to JSON, we recommend that you configure your function's log level using Lambda Advanced Logging Controls and not by setting the log level in code.